ivadomed/transforms.py

```diff
@@ -142,23 +143,25 @@ def __call__(self, sample):
         input_data = sample['input']
         roi_data = sample['roi']
 
-        w, h = input_data[0].size
-        th, tw = self.size
-
-        th_half, tw_half = int(round(th / 2.)), int(round(tw / 2.))
-
         # compute center of mass of the ROI
         x_roi, y_roi = center_of_mass(np.array(roi_data).astype(np.int))
         x_roi, y_roi = int(round(x_roi)), int(round(y_roi))
 
-        # compute top left corner of the crop area
-        fh = y_roi - th_half
-        fw = x_roi - tw_half
+        for i in range(len(input_data)):
+            w, h = input_data[i].size
+            th, tw = self.size
+            th_half, tw_half = int(round(th / 2.)), int(round(tw / 2.))
 
-        params = (fh, fw, h, w)
-        self.propagate_params(sample, params, 0)
+            # compute top left corner of the crop area
+            fh = y_roi - th_half
+            fw = x_roi - tw_half
+            params = (fh, fw, w, h)
```

Comment:
Error comes from here! w and h have been swapped compared to `master`

Reply:
Wow, good catch! Thx @charleygros